src/Core/Logging/Extensions/ILogServiceExtensions.Trace.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceExtensions
+    {
+        public static void Trace<TState>(
```

Comment:
Codacy found an issue: [Method has 5 parameters, which is greater than the 4 authorized.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5762132)